hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieCleanConfig.java

```diff
@@ -89,24 +89,7 @@ public class HoodieCleanConfig extends HoodieConfig {
           return Option.of(HoodieCleaningPolicy.KEEP_LATEST_FILE_VERSIONS.name());
         }
         return Option.empty();
-      })
-      .withDocumentation("Cleaning policy to be used. The cleaner service deletes older file "
```

Comment:
There are new docs added to this config.  Could you move them to the right place?